ovn_k8s/common/kubernetes.py

```diff
@@ -17,22 +17,60 @@
 
 import ovs.vlog
 
+from os import path
 from ovn_k8s.common import exceptions
+from ovn_k8s.common.util import ovs_vsctl
 
+CA_CERTIFICATE = "/usr/share/openvswitch/k8s-ca.crt"
 vlog = ovs.vlog.Vlog("kubernetes")
 
 
+def _get_api_params():
+    k8s_api_server = ovs_vsctl("--if-exists", "get", "Open_vSwitch", ".",
+                               "external_ids:k8s-api-server").strip('"')
+    if "https" in k8s_api_server:
+        k8s_ca_certificate = ovs_vsctl("--if-exists", "get", "Open_vSwitch",
+                                       ".", "external_ids:k8s-ca-certificate"
+                                       ).strip('"')
+        if k8s_ca_certificate:
+            k8s_ca_certificate = k8s_ca_certificate.replace("\\n", "\n")
+            if not path.isfile(CA_CERTIFICATE):
+                ca_file = open(CA_CERTIFICATE, 'w+')
+                ca_file.write(k8s_ca_certificate)
+            ca_cert = CA_CERTIFICATE
```

Comment:
I need some help understanding this logic.
It seems to me that you read an entire certificate from OVSDB and then store it in a hardcoded location /usr/share/openvswitch/k8s_ca_cert - then why not just let the operator store the certificate in that path and just verify that the path exists?

We can hardcode the path and not even store the ca_cert location in ovs db, or we can store a pointer to the filename in ovsdb. Maybe you have a good argument for storing the whole certificate there?